### DISCUSSION TITLE
query to MR-Base exceeded 300 seconds and timed out. Please simplify the query

### DISCUSSION THREAD

**Discussion by zincfingers89** [MRCIEU/TwoSampleMR#383] [Q&A]
Hi, Im trying to run TwoSampleMR in a terminal based R (not RStudio) in my university HPC. However I unfortunately keep hitting this issue:

> ao <- available_outcomes(access_token=NULL)
Error in api_query("gwasinfo", access_token = access_token) : 
  The query to MR-Base exceeded 300 seconds and timed out. Please simplify the query.

It works no problem on my own laptop. I have loaded the library dplyr and twosampleMR already. Running R version"bird hippie"

Does anyone have any suggestions for what is going wrong here? I have come across some other queries on github but no working solution yet.

cheers,
Katy

**Comment by jiangcongxin** [comment]
I have also encountered this problem. Have you solved it?

**Reply by kin-ton** [reply]
Using VPN can solved it

**Reply by remlapmot** [reply]
This is not the answer. You were probably just fortunate that there wasn't too much load from other users.

Please see this comment https://github.com/MRCIEU/TwoSampleMR/issues/422#issuecomment-1659627224

**Comment by cykky** [comment]
I have also encountered this problem. Have you solved it?

**Comment by kin-ton** [comment]
server is busy ,because it’s too much data to accumulate.

---- Replied Message ----
From cykky ***@***.***> Date 04/03/2024 09:40 To MRCIEU/TwoSampleMR ***@***.***> Cc kin-ton ***@***.***>,
Comment ***@***.***> Subject Re: [MRCIEU/TwoSampleMR] query to MR-Base exceeded 300 seconds and timed out. Please simplify the query (Discussion #383)  

I have also encountered this problem. Have you solved it?
